storage/internal/test/conformance/test.pb.go

```diff
@@ -30,597 +43,1345 @@ const (
 	UrlStyle_BUCKET_BOUND_HOSTNAME UrlStyle = 2
 )
 
-var UrlStyle_name = map[int32]string{
-	0: "PATH_STYLE",
-	1: "VIRTUAL_HOSTED_STYLE",
-	2: "BUCKET_BOUND_HOSTNAME",
-}
+// Enum value maps for UrlStyle.
+var (
+	UrlStyle_name = map[int32]string{
+		0: "PATH_STYLE",
+		1: "VIRTUAL_HOSTED_STYLE",
+		2: "BUCKET_BOUND_HOSTNAME",
+	}
+	UrlStyle_value = map[string]int32{
+		"PATH_STYLE":            0,
+		"VIRTUAL_HOSTED_STYLE":  1,
+		"BUCKET_BOUND_HOSTNAME": 2,
+	}
+)
 
-var UrlStyle_value = map[string]int32{
-	"PATH_STYLE":            0,
-	"VIRTUAL_HOSTED_STYLE":  1,
-	"BUCKET_BOUND_HOSTNAME": 2,
+func (x UrlStyle) Enum() *UrlStyle {
```

Comment:
Curious about these updates in the generated code-- do the signed URL tests still work as-is?